client/my-sites/plans/jetpack-plans/products-grid-i5/index.tsx

```diff
@@ -128,7 +146,11 @@ const ProductsGridI5: React.FC< ProductsGridProps > = ( {
 	return (
 		<Experiment name={ SWITCH_PLAN_SIDES_EXPERIMENT }>
 			<section className="products-grid-i5__section">
-				<h2 className="products-grid-i5__section-title">{ translate( 'Product Bundles' ) }</h2>
+				<h2 className="products-grid-i5__section-title">
+					{ POPULAR_PRODUCTS_OFFERING_VARIANT === tracksVariation
+						? translate( 'Most Popular' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Most popular' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/pricing-page-ab-test)